core/ui_events.js

```diff
@@ -113,5 +114,51 @@ Blockly.Events.Click.prototype.fromJson = function(json) {
   this.blockId = json['blockId'];
 };
 
+/**
+ * Class for a trashcan open event.
+ * @param {boolean=} opt_isOpen Whether the trashcan flyout is opening.
```

Comment:
Should this be 'is open'?

Reply:
Updated. I think 'is open' is easier to understand.